exporter.php

```diff
@@ -75,33 +75,33 @@
 $cats  = [];
 $tags  = [];
 $terms = [];
-if ( isset( $term ) && $term ) {
-	$cat  = get_term( $term['term_id'], 'category' );
-	$cats = array( $cat->term_id => $cat );
-	unset( $term, $cat );
-} elseif ( 'all' == $args['content'] ) {
-	$categories = (array) get_categories( array( 'get' => 'all' ) );
-	$tags       = (array) get_tags( array( 'get' => 'all' ) );
-
-	$custom_taxonomies = get_taxonomies( array( '_builtin' => false ) );
-	$custom_terms      = (array) get_terms( $custom_taxonomies, array( 'get' => 'all' ) );
-	while ( $cat = array_shift( $categories ) ) {
-		if ( 0 == $cat->parent || isset( $cats[ $cat->parent ] ) ) {
-			$cats[ $cat->term_id ] = $cat;
-		} else {
-			$categories[] = $cat;
-		}
-	}
-	while ( $t = array_shift( $custom_terms ) ) {
-		if ( 0 == $t->parent || isset( $terms[ $t->parent ] ) ) {
-			$terms[ $t->term_id ] = $t;
-		} else {
-			$custom_terms[] = $t;
-		}
-	}
-
-	unset( $categories, $custom_taxonomies, $custom_terms );
-}
+//if ( isset( $term ) && $term ) {
```

Comment:
Just a minor suggestion, to actually delete the unneeded code instead of commenting it, in order to keep the overall code more clean and readable. I understand that we may need to add taxonomies in CPP at some point, but that's why we have git, so can go back in history ;)